man/systemd.timer.xml

```diff
@@ -305,6 +305,25 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AlwaysRealtime=</varname></term>
```

Comment:
hmm, the name is confusing to me. Can we find something else?

DeferMultiElapse=yes/no or so?

DeferNext=yes/no?

DeferPending=yes/no?

or so?

Reply:
well, but we always already defer reactivation until after termination... it's mostly that we skip the first reactivation even more than that.

maybe DeferReactivation=

Reply:
Sounds good!